src/main/scala/com/leagueprojecto/api/services/MatchHistoryManager.scala

```diff
@@ -71,7 +72,7 @@ class MatchHistoryManager extends FSM[State, StateData] with ActorLogging {
       dbService ! SaveMatches(region, summonerId, matchDetails)
 
       val mergedMatches = matches ++ matchDetails.map(m => m.matchId -> Some(m))
-      val mergedMatchesSeq = mergedMatches.values.map(_.get).toSeq
+      val mergedMatchesSeq = mergedMatches.values.filter(_.isDefined).map(_.get).toSeq
```

Comment:
If I remember correctly, a map(_.get) already filters out None's, isn't that right?